From: Chuck Ebbert <cebbert@redhat.com>
Remove low_latency flag setting from nozomi and mxser drivers

The kernel oopses if this flag is set.

Signed-off-by: Chuck Ebbert <cebbert@redhat.com>

--- linux-2.6.30.noarch.orig/drivers/char/nozomi.c
+++ linux-2.6.30.noarch/drivers/char/nozomi.c
@@ -1591,8 +1591,6 @@ static int ntty_open(struct tty_struct *
 
 	/* Enable interrupt downlink for channel */
 	if (port->port.count == 1) {
-		/* FIXME: is this needed now ? */
-		tty->low_latency = 1;
 		tty->driver_data = port;
 		tty_port_tty_set(&port->port, tty);
 		DBG1("open: %d", port->token_dl);
--- linux-2.6.30.noarch.orig/drivers/char/mxser.c
+++ linux-2.6.30.noarch/drivers/char/mxser.c
@@ -1048,8 +1048,6 @@ static int mxser_open(struct tty_struct 
 	if (retval)
 		return retval;
 
-	/* unmark here for very high baud rate (ex. 921600 bps) used */
-	tty->low_latency = 1;
 	return 0;
 }
 
